apps/dh/api-dh/source/DataHub.WebApi/Modules/ProcessManager/Calculations/Client/CalculationsClient.cs

```diff
@@ -57,15 +57,14 @@ public async Task<IEnumerable<IOrchestrationInstanceTypedDto<ICalculation>>> Que
             var calculationTypes = input.CalculationTypes?.Where(x => x != CalculationType.ElectricalHeating);
             var calculationQuery = new CalculationQuery(userIdentity)
             {
-                LifecycleState = lifecycleState,
+                LifecycleStates = lifecycleStates,
                 TerminationState = terminationState,
                 CalculationTypes = calculationTypes?.Select(x => x.Unsafe_ToProcessManagerCalculationType()).ToArray(),
                 GridAreaCodes = input.GridAreaCodes,
                 PeriodStartDate = input.Period?.Start.ToDateTimeOffset(),
                 PeriodEndDate = input.Period?.End.ToDateTimeOffset(),
                 IsInternalCalculation = isInternalCalculation,
-                // TODO: If input.State == ProcessState.Scheduled, then we should also filter
-                // by ScheduledToRunAt(OrLater). This is not yet supported in the custom query.
+                ScheduledAtOrLater = input.State == ProcessState.Scheduled ? DateTime.UtcNow : null,
```

Comment:
this is not working properly, i believe it is because this check is inverted (should be `>=` instead `<=`): 
https://github.com/Energinet-DataHub/opengeh-process-manager/pull/190/files#diff-ee867592574554c5facd9f328bd753db99eae6291786309a5c289da1b1259f18R102